.github/workflows/release2.yaml

```diff
@@ -279,3 +313,30 @@ jobs:
     - name: Upload AAR to Maven Central
       run: ./gradlew uploadArchives
       working-directory: android
+
+  java:
+    if: github.event.pull_request.merged
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Decrypt PGP key
+      uses: IronCoreLabs/ironhide-actions/decrypt@v1
+      with:
+        keys: ${{ secrets.IRONHIDE_KEYS }}
+        input: .github/signing-key.asc.iron
+    - name: Import PGP key
+      run: gpg --batch --import .github/signing-key.asc
+    - uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 8
```

Comment:
Just want to double check that this is the version we want?

Reply:
@coltfred @clintfred is this correct?

Reply:
It shouldn't matter, as we have target and source set to 8 in the build file. Maybe we should run this on 11.

https://github.com/IronCoreLabs/ironoxide-java/blob/master/java/tests/build.sbt#L25 (for reference).

Reply:
see my comment

Reply:
Okay, will switch to 11.